src/assets/js/main.js

```diff
@@ -1,18 +1,21 @@
 /*
  * Polyfills
  */
-import './polyfills/object-fit';
```

Comment:
nit: your linter looks differently configured to the default. we probs dont want to lint some files like this and not others.